docs/en_US/NAS/Overview.md

```diff
@@ -25,11 +25,12 @@ One-shot algorithms run **standalone without nnictl**. Only the PyTorch version
 
 Here are some common dependencies to run the examples. PyTorch needs to be above 1.2 to use ``BoolTensor``.
 
-* NNI 1.2+
 * tensorboard
 * PyTorch 1.2+
 * git
 
+One-shot NAS can be visualized with our visualization tool. Learn more details [here](./Visualization.md).
+
 ## Supported Distributed NAS Algorithms
 
 |Name|Brief Introduction of Algorithm|
```

Comment:
add a section for NAS visualization above `Reference and Feedback`:

## NAS Visualization

To help users track the process and status of how the model is searched under specified search space, we developed a visualization tool. It visualizes search space as a super-net and shows importance of subnets and layers/operations, as well as how the importance changes along with the search process. Please refer to [the document of NAS visualization]() for how to use it.